frontend/package.json

```diff
@@ -2,41 +2,41 @@
     "name": "kjudge-frontend",
     "version": "0.7.0",
     "description": "Templates and others for kjudge.",
-    "main": "index.js",
     "repository": "https://github.com/natsukagami/kjudge",
     "author": "Natsu Kagami",
     "license": "AGPL-3.0",
     "private": true,
-    "devDependencies": {
-        "@fullhuman/postcss-purgecss": "^3.1.3",
-        "@tailwindcss/postcss7-compat": "^2.0.2",
-        "@types/humanize-duration": "^3.18.0",
-        "@types/node": "^14.14.14",
-        "@types/react": "^17.0.0",
-        "autoprefixer": "^9.7.5",
-        "fontsource-ibm-plex-mono": "^3.1.5",
-        "fontsource-muli": "^3.1.5",
-        "parcel-bundler": "^1.12.4",
-        "prettier": "2.2.1",
-        "tailwindcss": "npm:@tailwindcss/postcss7-compat",
-        "typescript": "^4.1.3"
+    "devDependencies": {        
+        "@fullhuman/postcss-purgecss": "5.0.0",
+        "@types/humanize-duration": "3.27.1",
+        "@types/node": "20.1.3",
+        "@types/react": "18.2.6",
+        "autoprefixer": "10.4.14",
+        "parcel": "2.8.3",
+        "postcss": "8.4.23",
+        "prettier": "2.8.8",
+        "tailwindcss": "3.3.2",
+        "typescript": "5.0.4"
```

Comment:
Why the fixed versions rather than `^` semver-compatible versions? This breaks `yarn upgrade`/`yarn upgradeInteractive`

Reply:
@natsukagami fixed in [22f52b2](https://github.com/natsukagami/kjudge/pull/80/commits/22f52b20f24805104ee1006b52d521e74a8484cb). I just think that pinning dependencies might be better (having another `node-ipc` won't be fun...).